UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -452,7 +446,13 @@ private static Mind SpawnMind(CharacterSheet character, bool NonImportantMind =
 		mind.Ghost();
 		mind.SetGhost(ghosty);
 		mind.CurrentCharacterSettings = character;
-
+		if (NonImportantMind == false)
+		{
+			AdminLogsManager.AddNewLog(
+			null,
+			$"A new mind has been created. Mind NetID: {mind.netId}",
```

Comment:
you want to add character name as well probably

Reply:
updated